diff --git a/example/test.ws b/example/test.ws
index 0affa55..1873888 100644
--- a/example/test.ws
+++ b/example/test.ws
@@ -1,6 +1,6 @@
 http {
 	include mime.types;
-	error_page 404 =404 /errorPages/404.html
+	error_page 404 =404 /errorPages/404.html;
 	error_page 502 =502 /errorPages/502.html;
 
 	server {
diff --git a/src/class/File/TmpFile.cpp b/src/class/File/TmpFile.cpp
index 84f55c4..6691ad2 100644
--- a/src/class/File/TmpFile.cpp
+++ b/src/class/File/TmpFile.cpp
@@ -46,9 +46,9 @@ std::string TmpFile::format(const std::string &path) {
 	std::string fileName;
 
 	if (pos == std::string::npos) {
-		return TMP_FILE_PREFIX + path;
+		return FILE_PREFIX + path;
 	} else {
-		return path.substr(0, pos + 1) + TMP_FILE_PREFIX + path.substr(pos + 1);
+		return path.substr(0, pos + 1) + FILE_PREFIX + path.substr(pos + 1);
 	}
 }
 
@@ -56,9 +56,9 @@ bool TmpFile::matchFormat(const std::string &path) {
 	size_t pos = path.find_last_of('/');
 
 	if (pos == std::string::npos) {
-		return path.compare(0, sizeof(TMP_FILE_PREFIX) - 1, TMP_FILE_PREFIX) == 0;
+		return path.compare(0, sizeof(FILE_PREFIX) - 1, FILE_PREFIX) == 0;
 	} else {
-		return path.compare(pos + 1, sizeof(TMP_FILE_PREFIX) - 1, TMP_FILE_PREFIX) == 0;
+		return path.compare(pos + 1, sizeof(FILE_PREFIX) - 1, FILE_PREFIX) == 0;
 	}
 }
 
diff --git a/src/class/File/TmpFile.hpp b/src/class/File/TmpFile.hpp
index 35fb508..18196c1 100644
--- a/src/class/File/TmpFile.hpp
+++ b/src/class/File/TmpFile.hpp
@@ -9,7 +9,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 
-#define TMP_FILE_PREFIX ".webserv_tmp_"
+#define FILE_PREFIX ".webserv_tmp_"
 
 class TmpFile : public File {
   private:
